Docker/php/remp.sh

```diff
@@ -12,6 +12,9 @@ then
     yarn install --no-bin-links
     chmod -R u+x node_modules
 
+    yarn link --cwd ../Package/remp
+    yarn link remp
+
     npm run | grep "all-dev"
```

Comment:
Btw - side note. when is this applicable / possible.
From what I gathered - npm isn't installed in the containers co the `npm run` cannot succeed ever.
?

Reply:
Seems like it used to be a depdendency of node/yarn before and we didn't install it explicitly. We'll address this separately.